src/ui/registration/components/handAnimation.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useRef, useEffect } from 'react'
+import { StyleSheet, View, Animated } from 'react-native'
+import { HandIcon, SplashIcon } from 'src/resources/index'
+
+interface Props {}
+
+const styles = StyleSheet.create({
+  handPosition: {
+    // TODO change to be % based
+    top: -29,
```

Comment:
Why does this need to be % based? Because magic numbers seem hack-y?

Reply:
I think as long as there's a comment explaining that this is the position where the hand fits in the sparkly bit, it should be fine.